assets/css/dashboard/new-pa-message/message-tables.scss

```diff
@@ -0,0 +1,83 @@
+.message-table {
```

Comment:
Sorry, should have mentioned this up front, but we're starting to migrate new CSS to use CSS modules as a way to simplify and reduce style leakage. This would be a good opportunity to use a module file for this new CSS, with explicitly imported classnames.